tests/sender/test_irc.py

```diff
@@ -17,7 +17,7 @@ async def _send_welcome():
         await asyncio.sleep(0.5)
         handler._client._dispatcher(transport, Event("welcome", "server", "user"))
 
-    asyncio.create_task(_send_welcome())
+    asyncio.create_task(_send_welcome())  # noqa: RUF006
```

Comment:
Maybe add a comment what RUF006 is about?

Reply:
Yep, will do. It's the issue about not storing references to tasks: https://beta.ruff.rs/docs/rules/asyncio-dangling-task/